tests/foreman/rhai/test_rhai.py

```diff
@@ -80,7 +79,7 @@ def test_client_registration_to_rhai(self):
             vm.get('/var/log/redhat-access-insights/redhat-access'
                    '-insights.log', './insights_client_registration.log')
 
-    def test_org_selection_for_rhai(self):
+    def test_negative_org_not_selected(self):
```

Comment:
I agree with you @jyejare.  But I think `test_negative_org_not_selected` more clearly explains the test in my opinion.